client/src/index.jsx

```diff
@@ -32,6 +34,11 @@ class App extends React.Component {
     $.post('/api/reviews/vote', { info }, this.updateVotes.bind(this));
   }
 
+  searchQuery(e) {
+    console.log(`search parameters: `, e);
+
+  }
```

Comment:
Is this presently used for anything other than debugging?

Reply:
Just placeholder, I think Josh told us to do everything up to writing the actual POST request.